.drone.star

```diff
@@ -2672,3 +2676,105 @@ def calculateTestsToRunBasedOnFilesChanged(ctx):
             },
         },
     ]
+
+def genericCache(name, action, mounts, cache_key):
+    rebuild = "false"
+    restore = "false"
+    if action == "rebuild":
+        rebuild = "true"
+        action = "rebuild"
+    else:
+        restore = "true"
+        action = "restore"
+
+    step = {
+        "name": "%s_%s" % (action, name),
+        "image": "meltwater/drone-cache:v1",
```

Comment:
In core we use:
`"image": "plugins/s3-cache:1",`

Is there a reason to be different here?

Reply:
I copied the cache functions 1:1 from oCIS. I think back than I chose meltwater/drone-cache over plugins/s3-cache because it has more functionality. Eg. it creates an archive from the cache and only uploads this one. This can be an performance improvement in some situations. 
I already had a look if we can use these caching functions for the oCIS binary caching but I did not success because we don't have the OCIS_COMMIT_ID available in starlark...

Reply:
OK - I hadn't looked at the oCIS starlark.